### PR TITLE
Jasmeen/fix comm forum btn 386

### PR DESCRIPTION
**Pull request by KAUR1984**
Fixes the issue #386.

**_Tasks completed:_** 
- [x] Added the `Community forum` button at the top right of the header.
- [x] Added the `ACCESS-NRI` button at the top right of the header.
- [x] Added the `Contribute` button at the top right of the header.  

> Note: This pr also includes some changes from the branch - https://github.com/ACCESS-Hive/access-hive.github.io/tree/dev/homepage which was merged just before the skillshare session. Very happy to remove any changes we don't need in the development branch @svenbuder @atteggiani, just assigning you guys as reviewers for this pr :) .

### CODE REVIEW

File: docs/model_evaluation/index.md [hunk 1]
```diff
@@ -2,14 +2,8 @@
...
 <!-- Model evaluation is about measuring how fit for purpose a particular model is.  -->
 
-## What is MED about?
-
 ACCESS-NRI's "Model Evaluation and Diagnostics" work is a critical facet of climate modeling, encompassing various tasks designed to ensure the model's reliability and accuracy.
 
-**Evaluation** involves scrutinizing the model through Model/Observation confrontations, checking its performance against real-world observations. It also includes experiment comparisons, testing the model under different scenarios, and inter-model comparisons like the Coupled Model Intercomparison Project (CMIP), assessing how the ACCESS-NRI model fares when compared with other climate models.
-
-**Diagnostics** involves constant monitoring of model runs to detect any anomalies or inconsistencies and a thorough analysis of outputs to verify the model's accuracy over time.
-
```
Comment (KAUR1984) [inline docs/model_evaluation/index.md:12]: Changes imported from https://github.com/ACCESS-Hive/access-hive.github.io/tree/dev/homepage

File: docs/model_evaluation/index.md [hunk 2]
```diff
@@ -75,14 +69,15 @@ If you are new to model evaluation and diagnostics, we recommend you read our [G
...
-* Model Diagnostics for on-the-fly analysis of your models at different snapshots
-* Data format processing tools like APP4
-* An Evaluation Recipe Gallery with searching functionality
+We are currently setting up a range of tools that will help you to better evaluate and diagnose climate models:  
+* Model Diagnostics for on-the-fly analysis of your models at different snapshots  
+* Data format processing tools like APP4  
+* An Evaluation Recipe Gallery with searching functionality  
 
-While we are working on these, we have collected a number of links to existing tools in our [community tab](../community_resources/index.md) (note that we are not currating them).
+While we are working on these, we have collected a number of links to existing tools in our [community tab](../community_resources/index.md) (note that we are not currating them).
```
Comment (KAUR1984) [inline docs/model_evaluation/index.md:80]: Previous changes imported from https://github.com/ACCESS-Hive/access-hive.github.io/tree/dev/homepage

File: docs/model_evaluation/index.md [hunk 3]
```diff
@@ -119,4 +114,10 @@ TBD: Add APP4 to navigation (replace **Model Format Processing**?)
...
 
--->
\ No newline at end of file
+-->
+
+## What is MED about?
+
+**Evaluation** involves scrutinizing the model through Model/Observation confrontations, checking its performance against real-world observations. It also includes experiment comparisons, testing the model under different scenarios, and inter-model comparisons like the Coupled Model Intercomparison Project (CMIP), assessing how the ACCESS-NRI model fares when compared with other climate models.
+
+**Diagnostics** involves constant monitoring of model runs to detect any anomalies or inconsistencies and a thorough analysis of outputs to verify the model's accuracy over time.
```
Comment (KAUR1984) [inline docs/model_evaluation/index.md:123]: Previous changes imported from https://github.com/ACCESS-Hive/access-hive.github.io/tree/dev/homepage

File: docs/model_evaluation/model_evaluation_getting_started/model_evaluation_getting_started.md [hunk 4]
```diff
@@ -1,6 +1,6 @@
-# Model Evaluation Environment at Gadi@NCI
+# Model Evaluation Environment at NCI's Gadi
 
-At this stage of *Getting Started*, we assume that you already have access to Gadi@NCI. If this is not the case, please go to our instructions on how to get [access to Gadi@NCI](./access_to_gadi_at_nci.md).
+At this stage of *Getting Started*, we assume that you already have access to Gadi@NCI. If this is not the case, please go to our instructions on how to get [access to NCI's Gadi](../../get_started).
```
Comment (KAUR1984) [inline docs/model_evaluation/model_evaluation_getting_started/model_evaluation_getting_started.md:3]: Previous changes imported from https://github.com/ACCESS-Hive/access-hive.github.io/tree/dev/homepage

File: overrides/partials/copyright.html [hunk 5]
```diff
@@ -1,3 +1,22 @@
...
+  all copies or substantial portions of the Software.
+  THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+  IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+  FITNESS FOR A PARTICULAR PURPOSE AND NON-INFRINGEMENT. IN NO EVENT SHALL THE
+  AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+  LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING
+  FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS
+  IN THE SOFTWARE.
+-->
+
```
Comment (KAUR1984) [inline overrides/partials/copyright.html:19]: Added acknowledgement notices at the top of `partials` files.

File: overrides/partials/header.html [hunk 6]
```diff
@@ -0,0 +1,129 @@
...
...1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
```
Comment (KAUR1984) [inline overrides/partials/header.html:114]: Fontawesome github icon didn't seem to work for me when used as follows: 

```css
<i class="fa-brands fa-github">
```

> Very much down to suggestions from the team to import the icon, other than as a `svg` file.

**Review by svenbuder — APPROVED** [review]
Happy with the changes.
@KAUR1984 Just to doublecheck: do you want to merge with development or dev/homepage?

**Comment by KAUR1984** [comment]
Thanks heaps @svenbuder ! 

I was hoping to merge this in the `development` branch as I have removed some of the changes of `dev/homepage` which were written just before the skill share and might need some tuning. I have flagged the `dev/homepage` changes in the comments above ^, these changes need to be reviewed if they could be merged in `development`. 

Or I am happy to remove all the changes of `dev/homepage` branch if you guys prefer :).

**Comment by svenbuder** [comment]
> > Happy with the changes.
> > @KAUR1984 Just to doublecheck: do you want to merge with development or dev/homepage?
> 
> Thanks heaps @svenbuder !
> 
> I was hoping to merge this in the `development` branch as I have removed some of the changes of `dev/homepage` which were written just before the skill share and might need some tuning. I have flagged the `dev/homepage` changes in the comments above ^, these changes need to be reviewed if they could be merged in `development`.
> 
> Or I am happy to remove all the changes of `dev/homepage` branch if you guys prefer :).

Great. That absolutely makes sense. Just wanted to avoid the merging into the wrong branch :)

**Comment by atteggiani** [comment]
Pushed a few minor changes:
- Removed "white-text" class, as not needed. The text inside the banner is already white (not properly white but equal to var(--md-primary-bg-color).

- Fixed fontawesome icons (had to import all.css in "mkdocs.yaml") and reverted back to `<i class="fa-brands fa-github">`. Also added `style="font-size: 1.3em"` to make the icons same size as before.

Everything else is Pretty good!

**Review by atteggiani — APPROVED** [review]

**Comment by KAUR1984** [comment]
Awesome! Thanks for making the fixes @atteggiani ! Will just merge this pr then :)

**Pull request merged**